src/super_gradients/recipes/imagenet_mobilenetv3_small.yaml

```diff
@@ -0,0 +1,21 @@
+# MobileNetV3 Small Imagenet classification training:
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.